src/main/java/org/opensearch/knn/jni/FaissService.java

```diff
@@ -463,4 +463,8 @@ public static native KNNQueryResult[] rangeSearchIndex(
         int indexMaxResultWindow,
         int[] parentIds
     );
+
+    public static native void setMergeInterruptCallback();
```

Comment:
Can we remove these and just have one interrupt callback registered at the time of library initialization? I dont think these need to be in the interface.

Reply:
I just prefer not to do this when its a global, static callback.